pkg/cmd/server/kubernetes/master.go

```diff
@@ -97,6 +105,23 @@ func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string
 	recycler.Run()
 }
 
+// attemptToLoadRecycler tries decoding a pod from a filepath for use as a recycler for a volume.
+// If a path is not set as a CLI flag, no load will be attempted and no error returned.
+// If a path is set and the pod was successfully loaded, the recycler pod will be set on the config and no error returned.
+// Any failed attempt to load the recycler pod will return an error.
+func attemptToLoadRecycler(path string, config *volume.VolumeConfig) error {
```

Comment:
Noticed that this is the same method (but with logging) used in the kube controller manager's plugins https://github.com/openshift/origin/blob/master/Godeps/_workspace/src/k8s.io/kubernetes/cmd/kube-controller-manager/app/plugins.go#L75-75.

Exposing it as a util and reuse it?